boot/boot_test.go

```diff
@@ -197,7 +199,7 @@ func setupUC20Bootenv(c *C, bl bootloader.Bootloader, opts *bootenv20Setup) (res
 		// multiple modeenvs from a single test and just call the restore
 		// function in between the parts of the test that use different modeenvs
 		r := func() {
-			emptyModeenv := &boot.Modeenv{}
+			emptyModeenv := &boot.Modeenv{Mode: "run"}
```

Comment:
empty now sounds strange,  maybe @anonymouse64 has suggestions

Reply:
probably defaultModeenv is the right thing to call it

Reply:
Yeah, not so empty anymore.